Description: Use Digest::SHA from perl core instead of 
 Digest::SHA1.
 .
 See: http://bugs.debian.org/594273
Origin: vendor
Bug: http://rt.cpan.org/Ticket/Display.html?id=69811
Forwarded: yes
Author: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2011-08-12

--- a/t/030_with_checksum.t
+++ b/t/030_with_checksum.t
@@ -9,7 +9,7 @@
 
 use Test::Requires {
     'Digest' => 0.01, # skip all if not installed
-    'Digest::SHA1' => 0.01,
+    'Digest::SHA' => 0,
     'JSON::Any' => 0.01,
 };
 
--- a/lib/MooseX/Storage/Base/WithChecksum.pm
+++ b/lib/MooseX/Storage/Base/WithChecksum.pm
@@ -85,7 +85,7 @@
         }
     }
     else {
-        return Digest->new($d || "SHA1", @args);
+        return Digest->new($d || "SHA-1", @args);
     }
 }
 
